packages/replay/src/replay.ts

```diff
@@ -323,9 +330,18 @@ export class ReplayContainer implements ReplayContainerInterface {
    * from calling both `flush` and `_debouncedFlush`. Otherwise, there could be
    * cases of mulitple flushes happening closely together.
    */
-  public flushImmediate(): Promise<void> {
+  public async flushImmediate(waitForCompression?: boolean): Promise<void> {
```

Comment:
Should this default true?

Reply:
Hmm, I guess we could also remove this option and just always use this behavior 🤔 WDYT?

Reply:
That'd be even better